<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewHall</class>
 <widget class="QWidget" name="AddNewHall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>座位图：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>总座位：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_totalSeats">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>行  数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_row">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>列  数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_column">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电影院：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>种  类：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>141</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_cinema">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_type"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddNewHall</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
